internal/mode/static/nginx/config/execute_test.go

```diff
@@ -3,27 +3,25 @@ package config
 import (
 	"testing"
 
+	. "github.com/onsi/gomega"
+
 	"github.com/nginxinc/nginx-kubernetes-gateway/internal/mode/static/nginx/config/http"
 )
 
 func TestExecute(t *testing.T) {
 	defer func() {
-		if r := recover(); r != nil {
-			t.Errorf("execute() panicked with %v", r)
-		}
+		g := NewWithT(t)
+		g.Expect(recover()).Should(BeNil())
 	}()
-
+	g := NewWithT(t)
 	bytes := execute(serversTemplate, []http.Server{})
-	if len(bytes) == 0 {
-		t.Error("template.execute() did not generate anything")
-	}
+	g.Expect(len(bytes)).ToNot(Equal(0))
```

Comment:
You could also write this as:
```
g.Expect(len(bytes).ToNot(BeEmpty())
```

Either way works but I thought I'd mention it.

Reply:
I tried this out and it didn't end up working. From the Gomega docs: _BeEmpty succeeds if actual is empty. Actual must be of type string, array, map, chan, or slice._, so seems like it doesn't work in this case because `len(bytes)` returns an int.

Reply:
My bad, I forgot to remove the len(). This should work:
```
g.Expect(bytes).ToNot(BeEmpty())
```